cartography/data/jobs/cleanup/aws_import_tgw_cleanup.json

```diff
@@ -0,0 +1,24 @@
+{
+  "statements": [
+  {
+    "query": "MATCH (n:AWSTransitGateway)-[r:RESOURCE|:SHARED_WITH]-() WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DETACH DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
[consider] For readability I think it's better to be explicit and match on specific node labels rather than `()`.

I won't block on this though because there's no perf impact and the code isn't doing something that'll create bugs down the road like what I call an "unlabeled delete" where you do something like `match (n)-[:SOME_REL]->(:SomeOtherLabel) detach delete n`. We've gotten really hard to trace bugs due to this in the past.

Anyway, just something to consider.

Reply:
Good point. Those single-line queries were just getting really long. I'd even consider switching to YAML for the cleanup queries, so you could take advantage of line folding:
```yaml
statements:
- query: |
    MATCH (n:AWSTransitGateway)-[r:RESOURCE|:SHARED_WITH]-()
    WHERE r.lastupdated <> {UPDATE_TAG} 
    WITH r 
    LIMIT {LIMIT_SIZE} DETACH DELETE (r) return COUNT(*) as TotalCompleted
  iterative: true
  iteration_size: 100
```